modules/references/client/components/create-reference/Info.js

```diff
@@ -19,24 +21,24 @@ UserLink.propTypes = {
 /**
  * Error message when trying to give a reference to oneself.
  */
-export function ReferenceToSelfInfo() {
-  return (<div className="alert alert-warning">Sorry, you can&apos;t give a reference to yourself.</div>);
-}
+export const ReferenceToSelfInfo = withNamespaces('reference')(function ({ t }) {
+  return (<div className="alert alert-warning">{t('Sorry, you can\'t give a reference to yourself.')}</div>);
```

Comment:
I wonder if `\'` will end up to strings and if we could use `"` here:
```js
t("Sorry, you can't give a reference to yourself.")
```

Reply:
This will produce a linter warning:

```
warning  Strings must use singlequote  quotes
```

It seems that the `\` doesn't become a part of the translation keys. (At least they're logged without `\` when translation debugger is on.)

I'd prefer to leave this as it is for now (more consistency in quotes, personal bias) and if it ever causes issues, it can be replaced.